packages/mobile/src/firebase/notifications.ts

```diff
@@ -82,11 +82,18 @@ export const handleNotification = (
   }
   switch (notification.data.type) {
     case NotificationTypes.PAYMENT_REQUESTED:
-      dispatch(handlePaymentRequested(notification.data, notificationState))
+      dispatch(
+        handlePaymentRequested((notification.data as unknown) as PaymentRequest, notificationState)
```

Comment:
why the cast to `unknown` first?

Reply:
typescript requires it since there is no type intersection (something like that)